### ISSUE TITLE
Serializer Configuration [proposed Label] feature

### ISSUE THREAD

**Issue by pgaref** [raulcf/SEEPng#61]
Currently supported SEEP serializers are:
- Java (default)
- Kryo
- Avro

First in the TODO list is the user to be able to select the preferred serializer. For this reason, we need to add extra configuration option in the MasterConfig and WorkerConfig - we also need to check those two are consistent when workers connect to the master(?) 

Second: when users implement custom classes for the queries there should be an easy way to register those classes to the serializer. Currently those classes should be part of the seep-common project in order to be serializable by both the master and the workers. Which is the easiest way to do that?

**Comment by raulcf** [comment]
Sounds good. However, I'd reduce the priority of this until there is a use case that requires a specific serializer. Do you have one already?

**Comment by pgaref** [comment]
I have done some progress in a local branch but nothing complete  - I was also wondering if the same serializers should be used for sending SeepCommands over network or keee using Kryo for that..(maybe this makes things more complicated since the nodes should know which serialiser to use beforehand - to send the bootstrap command)
I totally agree with the priority - its not urgent.

**Comment by raulcf** [comment]
right, it should only be implemented under demand. Once someone has a good use case for this. I left this in the code to show the flexibility and open that door.
thanks!
